### DISCUSSION TITLE
Hosting Team meeting agenda 2024-12-11

### DISCUSSION THREAD

**Discussion by Crixu** [WordPress/hosting-handbook#281] [Weekly Meetings]
Hosting Team meeting agenda 2024-12-04

---

PUBLIC AGENDA URL:

PUBLIC SUMMARY URL (if needed): 

# Agenda

WordPress Community
- WordPress Core news

Hosting Team Time
- Hosting Team Reps for 2025

Hosting Team To-Do

Open Floor
- Anything not in the agenda

Closing
- Next Meeting Times

# Meeting

/here [meeting type="hosting"] Who's here to chat about hosting?

👋 A warm welcome to all WordPress hosting enthusiasts! We're here to dive into the exciting world of WordPress hosting and development. Have questions? Looking for more information? Check out [Make/Hosting](https://make.wordpress.org/hosting/) for comprehensive guides and updates.

🥇 Participants in the [Five for the Future](https://wordpress.org/five-for-the-future/) program, we'd love to recognize your contributions! Please share your WordPress.org username in a thread here 👉

🤗 Newcomers, we're thrilled to have you on board! You are welcome to introduce yourself and share your experience or what you hope to contribute to WordPress. 👉

_Hello, I am YOURNAME , and today I am here leading this meeting. If you have any questions or queries, please do not hesitate to ask. At the end of the agenda, we have an open time to discuss anything else you want [about the team's projects](https://make.wordpress.org/hosting/handbook/get-involved/team-projects/)._

## Agenda

🟢 Here is the agenda for this meeting

[…] Use the agenda above […]

Would anyone like to add to or change any items on the agenda?

OK, let's get started with the WordPress Community news.

### 🟢 WordPress Core news

We are currently in the [WordPress 6.8 release cycle](https://make.wordpress.org/core/6-8/). The [call for volunteers for the 6.8 release squad](https://make.wordpress.org/core/2024/11/21/wordpress-6-8-call-for-volunteers/) is open until Dec 6. There is also a [proposal open for the major releases for 2025](https://make.wordpress.org/core/2024/11/19/proposal-major-releases-for-2025/).

- [What’s new in Gutenberg 19.8? (04 December)](https://make.wordpress.org/core/2024/12/04/whats-new-in-gutenberg-19-8-4-december/)

Ok, let's move on to Hosting Team time.

## 🟩 Open Floor

Ok, if nothing else, let's move on to the Open Floor. You are welcome to bring any topic up for discussion at this time.

…

### 🟩 Meeting Times

- 2024-12-18 1800 UTC

### Goodbye

That wraps up today's meeting. Please keep in mind, our discussions can continue in the [#hosting Slack channel](https://wordpress.slack.com/archives/C3D6T7F8Q) or on [Make/Hosting](https://make.wordpress.org/hosting/) anytime.

I hope you have a wonderful rest of your week! [/meeting]

**Comment by Crixu** [comment]
## Hosting Team Time

🟢 WordPress Hosting Team Reps
[Voting for Hosting Team Reps 2025](https://make.wordpress.org/hosting/2024/11/29/voting-for-hosting-team-reps-2023-2/)
As announced we closed the survey earlier today to have enough time to prepare the results for the meeting. Best news so far is that we got only got positive responses on both the candidates. Congrats again to @Crixu and @Zunaid for this awesome result. 
This means the next steps involve:

- Checking in with the candidates on their final availability
- Announce the Hosting Team Reps for 2025
- Start the transition to be ready for 2025

🟢 WordCamp Asia Contributor Day
On Monday Jhimross reached out and asked for Volunteers to lead the Hosting Table at the upcoming WordCamp Asia 2025. @ Wes volunteered to take this role and will be the table lead for WC Asia 2025. Any special topics the team there wants to tackle? 
Who of the team will be around for WC Asias Contributor Day? 

🟢 Open work
We do have some open tickets in the hosting handbook that require a review and look like first great tasks if you want to contribute to the team:
https://github.com/WordPress/hosting-handbook/pulls
 
For the [Test Runner](https://github.com/WordPress/phpunit-test-runner/pulls) and the [Test Reporter ](https://github.com/WordPress/phpunit-test-reporter/pulls)we also do have some open PRs and issues!

**Comment by chaion07** [comment]
🟢 WordPress Core news

We are currently in the [WordPress 6.8 release cycle](https://make.wordpress.org/core/6-8/). The [call for volunteers for the 6.8 release squad](https://make.wordpress.org/core/2024/11/21/wordpress-6-8-call-for-volunteers/) is open until Dec 6. There is also a [proposal open for the major releases for 2025](https://make.wordpress.org/core/2024/11/19/proposal-major-releases-for-2025/).

- [WordPress 6.7 Performance Improvements](https://make.wordpress.org/core/2024/12/10/wordpress-6-7-performance-improvements/)
- [WordPress 6.7 Release Retrospective](https://make.wordpress.org/core/2024/12/06/wordpress-6-7-release-retrospective/)
- [Hallway Hangout: Performance End of Year Review 2024](https://make.wordpress.org/core/2024/12/06/hallway-hangout-performance-end-of-year-review-2024/)
- [What’s new in Gutenberg 19.8? (04 December)](https://make.wordpress.org/core/2024/12/04/whats-new-in-gutenberg-19-8-4-december/)
- [Let’s reconsider adopting the WP Consent API](https://make.wordpress.org/core/2024/12/04/lets-reconsider-adopting-the-wp-consent-api/)
- [The Photos Directory gets a refresh and is now powered by blocks](https://make.wordpress.org/meta/2024/12/10/the-photos-directory-gets-a-refresh-and-is-now-powered-by-blocks/)

**Comment by Crixu** [comment]
## Community News

### State of the Word 2024
Don't miss the [State Of The Word](https://wordpress.org/state-of-the-word/) which takes place on Monday, Dec 16th in Tokyo, Japan. There will be a [lot of Watch Parties](https://events.wordpress.org/?s=State+of+the+word) you can join virtually or in person!.

**Reply by chaion07** [reply]
I believe the following should also be added to the community news

- [Meetup Organizer Newsletter: December 2024](https://make.wordpress.org/community/2024/12/06/meetup-organizer-newsletter-december-2024/)
- [Starting – and continuing – a conversation about Indigenous representation and perspectives in WordPress](https://make.wordpress.org/community/2024/11/26/starting-and-continuing-a-conversation-about-indigenous-representation-and-perspectives-in-wordpress/)
